ISSUE_TEMPLATE/2-feature-request.md

```diff
@@ -0,0 +1,30 @@
+---
+name: "🚀 Feature Request"
+about: "Suggest an idea or new feature"
+title: ""
+labels: "Type: Feature"
+---
+
+## 🚀 Feature Request
+
+## Expected improvement
+<!-- Add a clear and concise description of the under-optimal behavior. Eg. "Such and so happens when (...), which isn't ideal because (...)" -->
+<!-- Is your feature request related to a specific problem? If so, please describe it.-->
+*
+
+## Sought-for solution
+<!-- A clear and concise description of the expected behavior of the feature -->
+*
+
+## Alternatives considered
+<!-- Optional: A clear and concise description of any alternative solutions or features you've considered. -->
+*
+
+## Acceptance criteria
+<!-- List the criteria the solution needs to be deemed acceptable -->
+*
+
+## Additional context and visual reference
+<!-- Optional: Add any other context or additional information about the problem here.
+If applicable, add visual reference here-->
+*
```

Comment:
I don't think these feel like "Feature" requests, more like "enhancements", which are already covered in "4-enhancement_request.md". I think a rewording of these points to be more in line with an actual new feature in the app would be better

Reply:
I think it's natural for a feature request to be similar to an enhancement request - a feature is a sort of enhancement.
The idea is: features are a common and prominent enough enhancement to warrant their own request marker (besides laying the issue's structure, the template helps labeling it).

But it's not clear to me how I could better differentiate the two. @AmadeusK525, if you (or any other person) has some idea on this, feel free to share.

Otherwise, if you people find it preferable, we could delete the feature-request and keep only the enhancements template.

Reply:
I agree with that, I think we should have just one template for those two

Reply:
How about something like this? I agree with you @ericlesaquiles in the sense that there's enough difference between a feature request and an enhancement request to warrant different templates
```markdown
## Description
<!-- Add a clear and concise description of what this new feature will do and what it aims to achieve -->
<!-- Is this new feature related to a specific problem? If so, please describe it -->

## Implementation details
<!-- How will this feature work? Do you already have something in mind? A potential implementation may be
explained here, in a concise way -->

## Potential caveats
<!-- Is there something the programmer should look out for while implementing this new feature? Some potential
problems that may arise while doing so? Please describe them here, if any -->

## Acceptance criteria
<!-- A straightforward bullet list detailing what needs to be there for this task to be considered 'done'. All
points should be very direct and clear -->

## Additional context and visual reference
<!-- Any more general comments about the issue? Feel free to put them here. You can add some visual
reference as well, when applicable, to aid in the development of this feature and help the developer
visualize what they should be implementing -->
```

Reply:
LGTM!

Reply:
I think features and enhancements are very close in meaning, but we can have both without any impairment.  @AmadeusK525 your propose looks nice!